public/app/plugins/datasource/azuremonitor/azure_log_analytics/azure_log_analytics_datasource.test.ts

```diff
@@ -129,37 +139,6 @@ describe('AzureLogAnalyticsDatasource', () => {
     });
   });
 
-  describe('When performing targetContainsTemplate', () => {
-    it('should return false when no variable is being used', () => {
-      const query = createMockQuery();
-      const ds = new AzureMonitorDatasource(ctx.instanceSettings);
-      query.queryType = AzureQueryType.LogAnalytics;
-      expect(ds.targetContainsTemplate(query)).toEqual(false);
-    });
-
-    it('should return true when resource field is using a variable', () => {
-      const templateSrv = new TemplateSrv();
-      const query = createMockQuery();
-      templateSrv.init([singleVariable]);
-
-      const ds = new AzureMonitorDatasource(ctx.instanceSettings, templateSrv);
-      query.queryType = AzureQueryType.LogAnalytics;
-      query.azureLogAnalytics = { resources: [`$${singleVariable.name}`] };
-      expect(ds.targetContainsTemplate(query)).toEqual(true);
-    });
-
-    it('should return false when a variable is used in a different part of the query', () => {
-      const templateSrv = new TemplateSrv();
-      const query = createMockQuery();
-      templateSrv.init([singleVariable]);
-
-      const ds = new AzureMonitorDatasource(ctx.instanceSettings, templateSrv);
-      query.queryType = AzureQueryType.LogAnalytics;
-      query.azureResourceGraph = { query: `$${singleVariable.name}` };
-      expect(ds.targetContainsTemplate(query)).toEqual(false);
-    });
-  });
```

Comment:
I moved these tests to the datasourse tests. I don't think we need to test template vars here. WDYT?

Reply:
I think this is fine 👍